docs/release-notes.md

```diff
@@ -4,17 +4,28 @@ nav_order: 9
 
 # Release Notes
 
-## Upcoming Ignition 2.17.0 (unreleased)
+## Upcoming Ignition 2.17.1 (unreleased)
```

Comment:
This should be a full version jump aka 2.18.0

We use [semantic](https://www.geeksforgeeks.org/introduction-semantic-versioning/) versioning so x.y.z 

x = major version, reserved for breaking changes 
y = minor version, normal additive changes and such
z = patch version, usually small and needed to fix something which is not working as expected.